jest/jest-tests.ts

```diff
@@ -199,6 +199,16 @@ describe('Assymetric matchers', function () {
         }));
 
         expect.assertions(4);
+
+        interface Test {
+            a: number;
+            b: string;
+        }
+
+        const test: Test = {
```

Comment:
Yes, it is. It's useful to construct expected object outside of test call - this way you can assign type and property renaming in this type will also affects tests.